package.json

```diff
@@ -87,38 +85,28 @@
 		"@deque/dot": "^1.1.5",
 		"aria-query": "^3.0.0",
 		"axios": "^0.19.0",
-		"babelify": "^10.0.0",
-		"blanket": "~1.2.3",
-		"browserify": "^16.2.3",
 		"chai": "~4.2.0",
 		"chalk": "^4.1.0",
 		"chromedriver": "^87.0.0",
-		"clone": "~2.1.1",
+		"clone": "^2.1.2",
 		"conventional-commits-parser": "^3.1.0",
 		"css-selector-parser": "^1.3.0",
-		"derequire": "^2.1.1",
 		"emoji-regex": "8.0.0",
 		"es6-promise": "^4.2.6",
 		"esbuild": "^0.7.19",
 		"eslint": "^6.1.0",
 		"eslint-config-prettier": "^6.2.0",
 		"execa": "^2.0.2",
-		"fs-extra": "^8.0.1",
 		"globby": "^10.0.0",
 		"grunt": "^1.0.3",
 		"grunt-babel": "^8.0.0",
 		"grunt-bytesize": "^0.2.0",
 		"grunt-contrib-clean": "^2.0.0",
 		"grunt-contrib-concat": "^1.0.1",
-		"grunt-contrib-connect": "^2.0.0",
-		"grunt-contrib-copy": "^1.0.0",
 		"grunt-contrib-uglify": "^4.0.0",
 		"grunt-contrib-watch": "^1.1.0",
-		"grunt-parallel": "^0.5.1",
-		"grunt-run": "^0.8.1",
 		"html-entities": "^1.2.0",
 		"http-server": "^0.12.3",
-		"husky": "^3.0.0",
```

Comment:
Why are you removing this? Husky is what manages our Git hooks (automatically running Prettier, etc).

Reply:
I wondered what that did. I did a search of all the packages and this wasn't used in a require anywhere.

Reply:
I think `lint-staged` runs our prettier. It doesn't look like Husky is used in our `package.json`

Reply:
`husky` is what runs `lint-staged`. When installed, it sets up our Git hooks.

Reply:
We determined that we still need husky but it is not properly configured in our package.json (which explains how prettier isn't run on our commits sometimes).